README.md

```diff
@@ -107,6 +107,7 @@ library archives (`.a`).
 
 [^] On Debian/Ubuntu `libgtest-dev` only includes sources and headers. You must
 build the library binary manually. This can be done with the following command ```sudo apt-get install libgtest-dev && cd /usr/src/gtest && sudo cmake . && sudo make && sudo mv libg* /usr/lib/ ```
+[^^] OpenSSL 1.1.0 is default version of OpenSSL on Ubuntu 18.04 LTS Bionic. However, OpenSSL 1.1.0 is not supported by project now (since in the OpenSSL 1.1.0 were renamed or removed some interfaces), so be sure that you use OpenSSL 1.0.2 or earlier. To install OpenSSL 1.0.2 on Ubuntu 18.04 use `libssl1.0-dev` package.
```

Comment:
OpenSSL 1.1.0 is a default version of OpenSSL package for Ubuntu 18.04 LTS Bionic. However, OpenSSL 1.1.0 is not supported by the project as of now (some interfaces have been renamed or removed in version 1.1.0), so be sure you use OpenSSL 1.0.2 or earlier.

Reply:
Fixed